front-end/src/ui-components/TopicTag.tsx

```diff
@@ -0,0 +1,61 @@
+// Copyright 2019-2020 @paritytech/polkassembly authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import React, { ReactNode } from 'react';
+import { Label } from 'semantic-ui-react';
+import styled from '@xstyled/styled-components';
+
+interface Props{
+	children?: ReactNode,
+	className?: string,
+	content?: string,
+	topic: string
+}
+
+const TopicTag = ({ children, className, content, topic }: Props) => {
+
+	return (
+		<Label
+			className={className}
+			content={content}
+			topic={topic}
+		>
```

Comment:
you can when removing children, you can use a self closing tag `<Label ... />` and you won't need any `</Laber>` then.